test/pubsub.js

```diff
@@ -127,7 +126,7 @@ describe.skip('pubsub', function () {
 
       it('should exchange binary data', function () {
         const data = uint8ArrayFromString('a36161636179656162830103056164a16466666666f400010203040506070809', 'base16')
-        const topic = 'pubsub-binary'
+        const topic = 'pubsub\nbinary\n'
```

Comment:
Same, this should be a separate test - we are testing binary data here, not newlines in topic names.